.github/workflows/deploy-on-release-to-dot-org.yml

```
@@ -21,6 +21,9 @@ jobs:
           php-version: '7.4'
           tools: composer:v2
           coverage: none
+          github-token: ${{ secrets.PUBLIC_REPO_ACCESS }}
```

actionlint (1.7.7)

input "github-token" is not defined in action "shivammathur/setup-php@v2". available inputs are "coverage", "extensions", "ini-file", "ini-values", "php-version", "php-version-file", "tools"

(action)